### ISSUE TITLE
[Question] How to pass custom error message to custom matchers in Playwright

### ISSUE THREAD

**Issue by sankarkec** [microsoft/playwright#22453]
I am creating my own custom matches in playwright. Everything works fine expect custom message. Below is my code

expect.extend({
toEqualIgnoreCase(actual: string, expected: string, messageOrOptions?: string) {
        console.log("actual: " + actual + ", expected: " + expected+ ", Message: " + messageOrOptions);
        if (actual.toUpperCase() == expected.toUpperCase()) {
            return {
                message: () =>  "Expected '" + actual + "' to equal ignore case '" + expected + "', '" + messageOrOptions + "'",
                pass: true,
            };
        } else {
            return {
                message: () =>  "Expected '" + actual + "' to equal ignore case '" + expected + "', '" + messageOrOptions + "'",
                pass: false,
            };
        }
    },
});

In Test i called like 

expect("test", "custom message").toEqualIgnoreCase("Test1");

But inside the custom matchers only actual and expected value received but custom message not received. How to pass the message for custom matchers.

**Comment by dgozman** [comment]
@sankarkec I don't think this is supported today. Let me keep this issue open as a feature request.

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

Thank you for your involvement. This issue was closed due to limited engagement (upvotes/activity), lack of recent activity, and insufficient actionability. To maintain a manageable database, we prioritize issues based on these factors.

If you have additional information not present in this issue that you think will help prioritizing it, please open a new issue and reference this one. More support or clarity on its necessity may prompt a review. Your understanding and cooperation are appreciated.
